lighthouse-core/test/gather/driver-test.js

```diff
@@ -702,7 +706,83 @@ describe('._waitForFcp', () => {
     await flushAllTimersAndMicrotasks();
     expect(waitPromise).toBeDone('Did not cancel promise');
     expect(driver.off).toHaveBeenCalled();
-    await expect(waitPromise).rejects.toMatchObject({message: 'Wait for FCP canceled'});
+    await expect(waitPromise).rejects.toMatchObject(
+      {message: 'Wait for "firstContentfulPaint" event canceled'}
+    );
+  });
+});
+
+describe('._waitForLcp', () => {
+  it('should not resolve until LCP fires', async () => {
```

Comment:
These test cases are pretty much a copy/paste from `_waitForFcp` cases above.  Would it make more sense to combine these into a section for `_waitForLifecycleEvent` and add test case to differentiate FCP and LCP?